Source/WebCore/dom/FullscreenManager.cpp

```diff
@@ -378,6 +380,18 @@ void FullscreenManager::exitFullscreen(RefPtr<DeferredPromise>&& promise)
             return;
         }
 
+        // Run the "fully unlock the screen orientation" steps with doc.
+        // https://w3c.github.io/screen-orientation/#dfn-fully-unlock-the-screen-orientation-steps
+        if (auto orientationManager = page->screenOrientationManager()) {
```

Comment:
We should split code changes out from test changes. The tests are changing so much in this PR, it is difficult to see what the progression from this change is.

Reply:
Yeah, my apologies for that. This was more for discussion. My intent was to split this into several PRs. 

I'll do that now.

Reply:
Oh, I see. Bri really wanted to land this and asked me to review. I didn't realize this wasn't ready.